src/assets/i18n/en.json

```diff
@@ -1,6 +1,8 @@
 {
   "API": {
     "BALANCE_TOO_LOW": "Balance too low for transaction",
+    "BALANCE_TOO_LOW_DETAIL": "Not enough {{ token }} on your account! The transaction costs are: {{ totalAmount }} ({{ amount }} + {{ fee }} fee). However your account balance is only {{ balance }}!",
```

Comment:
Hmm, the detailing is great but it's so long for the mobile

Reply:
It looks like this, if it helps your decision:
![screenshot_20180129-210851](https://user-images.githubusercontent.com/1086065/35531918-c325a1e4-0538-11e8-89c0-f026e80b35f6.jpg)

Reply:
Oh, I thought it was a toast. Seems great! 👍🏻

Reply:
Just realised it can also be a toast (forgot about it already):
![screenshot_20180131-211729](https://user-images.githubusercontent.com/1086065/35645483-4226270e-06cc-11e8-8230-0a2aca343049.jpg)
(important is, that the volume stuff is on the top :P)